### PR TITLE
Update toolchains

### PR DESCRIPTION
**Pull request by WilliamJamieson**
Brings asdf-standard tool configurations in line with asdf, PR: asdf-format/asdf#1091.

### CODE REVIEW

File: bandit.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+exclude_dirs:
+  - .eggs
+  - .git
+  - .pytest_cache
+  - .tox
+  - reference_files
+  - tests
+  - build
+  - compatibility_tests
```
Comment (eslavich) [inline bandit.yaml:11]: This directory doesn't exist for asdf-standard

File: pyproject.toml [hunk 2]
```diff
@@ -7,3 +7,21 @@ write_to = "src/asdf_standard/_version.py"
...
+force-exclude = '''
+^/(
+  (
+      \.eggs
+    | \.git
+    | \.pytest_cache
+    | \.tox
+    | reference_files
+    | build
+    | dist
```
Comment (eslavich) [inline pyproject.toml:19]: Should we be excluding the resources directory here?

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -11,28 +11,24 @@ repos:
...
+  rev: 22.1.0
+  hooks:
+    - id: black
 
 - repo: https://gitlab.com/pycqa/flake8
   rev: 4.0.1
   hooks:
     - id: flake8
-      args: ["--count", "--ignore", "E501, E203, W503"]
       exclude: "reference_files"
```
[the remark this answers is not in the document]
Comment (WilliamJamieson) [reply .pre-commit-config.yaml:28]: There should not be any python files in resources, so flake8 should not do anything there. The configuration is in the setup.cfg as it was before.

File: bandit.yaml (hunk 1 above)
Comment (WilliamJamieson) [reply bandit.yaml]: Its just yaml files so it should be ignored.

File: pyproject.toml (hunk 2 above)
Comment (WilliamJamieson) [reply pyproject.toml:19]: Again, resources is just yaml files so it will be ignored anyways.

**Pull request merged**